api/controllers/productController.js

```diff
@@ -0,0 +1,50 @@
+const { productService } = require('../services');
+const { CustomError } = require('../utils/error');
+const getProductByProductId = async (req, res) => {
+    try {
+        const { productId } = req.params;
+        if (isNaN(productId)) {
+            throw new CustomError('BAD_REQUEST', 400);
+        }
+        const product = await productService.getProductByProductId(productId);
+        res.status(200).json({ product: product });
```

Comment:
```suggestion
        res.status(200).json({ product });
```

아마 이렇게 작성하면 키 밸류 형태로 잘 들어갈겁니다 ㅎㅎ